recipes/fs.webdavfs/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "fs.webdavfs" %}
+{% set version = "0.2.1" %}
+{% set sha256 = "02a50cbb8107ebf5f04b44b36fb9413d2cb8eb54f84c1ac1d00e4ac034dd7bdf" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.zip
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - fs
+    - webdavclient2
+    - furl
+
+test:
+  imports:
+    - webdavfs.webdavfs.WebDAVFS
```

Comment:
You can't import `WebDAVFS` in this way since it's a class and not a module. I would just use:

```
imports:
   - webdavfs.webdavfs
   - webdavfs.opener
```

Reply:
OMG, Thx, fixing :)